src/vtbackend/Functions.h

```diff
@@ -618,10 +623,82 @@ inline auto const& functions() noexcept
     return funcs;
 }
 
+class SupportedSequences
+{
+  public:
+    SupportedSequences()
+    {
+        for (auto [index, seq]: crispy::indexed(functions()))
+        {
+            _supportedSequences[index] = seq;
+        }
+        crispy::sort(
+            _supportedSequences,
+            [](FunctionDefinition const& a, FunctionDefinition const& b) constexpr { return compare(a, b); });
+        _lastIndex = _supportedSequences.size();
+    }
+
+    [[nodiscard]] gsl::span<FunctionDefinition const> getAllSequences() const noexcept
+    {
+        return gsl::span<FunctionDefinition const>{_supportedSequences.cbegin(), _supportedSequences.cend()};
+    }
+
+    [[nodiscard]] gsl::span<FunctionDefinition const> getSupportedSequencesAtCurrentVTLevel() const noexcept
+    {
+        return gsl::span<FunctionDefinition const>{_supportedSequences.cbegin(),
+                                                   _supportedSequences.cbegin() + _lastIndex};
+    }
+
+    void updateVTLevel(VTType vt) noexcept
+    {
+        auto itr = std::partition(
+            _supportedSequences.begin(),
+            _supportedSequences.end(),
+            [vt](const FunctionDefinition& value) noexcept { return value.conformanceLevel <= vt; });
+
+        _lastIndex = std::distance(_supportedSequences.begin(), itr);
+        gsl::span<FunctionDefinition> availableDefinition{_supportedSequences.begin(), _lastIndex};
+        crispy::sort(
```

Comment:
C++20 finally provides a `constexpr` `std::sort()` implementation. We could try using that one, in case all platforms provide it.

Reply:
One of the ubuntu ci doesn't have new enough version so constexpr algorithms are not present there. But maybe I should also add another macro to conditionally add constexpr?

Reply:
Hm, maybe, but then use this macro only where we know it's not working otherwise, I'd suggest.

The place to put both macros would probably be best in `<crispy/defines.h>`